--- LVM2.2.02.130/conf/example.conf.in.orig	2015-09-10 22:32:38.524724874 +0100
+++ LVM2.2.02.130/conf/example.conf.in	2015-09-10 22:32:53.576726004 +0100
@@ -104,7 +104,7 @@
 	# Prefer the path with least value in lexicographical order.
 	# 
 	# Example
-	# preferred_names = [ "^/dev/mpath/", "^/dev/mapper/mpath", "^/dev/[hs]d" ]
+	preferred_names = [ "^/dev/mpath/", "^/dev/mapper/mpath", "^/dev/[hs]d" ]
 	# 
 	# This configuration option does not have a default value defined.
 
