diff -Naur LVM2.2.02.178/configure LVM2.2.02.178.tpg/configure
--- LVM2.2.02.178/configure	2018-06-13 12:32:34.000000000 +0000
+++ LVM2.2.02.178.tpg/configure	2018-07-03 21:02:13.690634950 +0000
@@ -9920,7 +9920,6 @@
 if [ `expr x"$CLVMD" : '.*cman.*'` != 0 ]; then
 	BUILDCMAN=yes
 	CLVMD_CMANAGERS="$CLVMD_CMANAGERS cman"
-	CLVMD_NEEDS_QDISKD=yes
 fi
 if [ `expr x"$CLVMD" : '.*corosync.*'` != 0 ]; then
 	BUILDCOROSYNC=yes
diff -Naur LVM2.2.02.178/configure.ac LVM2.2.02.178.tpg/configure.ac
--- LVM2.2.02.178/configure.ac	2018-06-13 12:32:34.000000000 +0000
+++ LVM2.2.02.178.tpg/configure.ac	2018-07-03 21:00:59.885639783 +0000
@@ -706,7 +706,6 @@
 if [[ `expr x"$CLVMD" : '.*cman.*'` != 0 ]]; then
 	BUILDCMAN=yes
 	CLVMD_CMANAGERS="$CLVMD_CMANAGERS cman"
-	CLVMD_NEEDS_QDISKD=yes
 fi
 if [[ `expr x"$CLVMD" : '.*corosync.*'` != 0 ]]; then
 	BUILDCOROSYNC=yes
