diff -up LVM2.2.03.01/libdm/dm-tools/Makefile.in.2~ LVM2.2.03.01/libdm/dm-tools/Makefile.in
--- LVM2.2.03.01/libdm/dm-tools/Makefile.in.2~	2018-11-07 19:50:51.416794058 +0100
+++ LVM2.2.03.01/libdm/dm-tools/Makefile.in	2018-11-07 19:50:51.413794044 +0100
@@ -44,7 +45,7 @@ dmsetup: dmsetup.o
 dmsetup.static: dmsetup.o $(INTERNAL_LIBS)
 	@echo "    [CC] $@"
 	$(Q) $(CC) $(CFLAGS) $(LDFLAGS) -static -L$(interfacebuilddir) \
-	      -o $@ $+ $(M_LIBS) $(PTHREAD_LIBS) $(STATIC_LIBS) $(LIBS)
+	      -o $@ $+ $(M_LIBS) $(PTHREAD_LIBS) $(STATIC_LIBS) $(LIBS) -L$(top_builddir)/libdm -ldevmapper -lm -pthread
 
 all: device-mapper
 
diff -up LVM2.2.03.01/libdm/Makefile.in.2~ LVM2.2.03.01/libdm/Makefile.in
--- LVM2.2.03.01/libdm/Makefile.in.2~	2018-11-07 19:50:51.409794026 +0100
+++ LVM2.2.03.01/libdm/Makefile.in	2018-11-07 19:50:51.405794007 +0100
@@ -42,11 +42,12 @@ INCLUDES = -I$(srcdir)/$(interface)
 
 ifeq ("@STATIC_LINK@", "yes")
 LIB_STATIC = $(interface)/libdevmapper.a
-endif
-
+TARGETS = libdevmapper.a
+else
 LIB_SHARED = $(interface)/libdevmapper.$(LIB_SUFFIX)
 LIB_VERSION = $(LIB_VERSION_DM)
 TARGETS = libdevmapper.$(LIB_SUFFIX) libdevmapper.$(LIB_SUFFIX).$(LIB_VERSION)
+endif
 
 CFLOW_LIST = $(SOURCES)
 CFLOW_LIST_TARGET = libdevmapper.cflow
@@ -67,6 +68,9 @@ dm-tools.device-mapper: $(TARGETS)
 libdevmapper.$(LIB_SUFFIX) libdevmapper.$(LIB_SUFFIX).$(LIB_VERSION): $(LIB_SHARED)
 	$(LN_S) -f $< $@
 
+libdevmapper.a: $(LIB_STATIC)
+	$(LN_S) -f $< $@
+
 .PHONY: install_dynamic install_static install_include \
 	install_ioctl install_ioctl_static \
 	install_pkgconfig
diff -up LVM2.2.03.01/tools/Makefile.in.2~ LVM2.2.03.01/tools/Makefile.in
--- LVM2.2.03.01/tools/Makefile.in.2~	2018-11-07 19:50:51.377793879 +0100
+++ LVM2.2.03.01/tools/Makefile.in	2018-11-07 20:04:43.322789539 +0100
@@ -138,7 +138,7 @@ man-generator: man-generator.o
 lvm.static: $(OBJECTS) lvm-static.o $(top_builddir)/lib/liblvm-internal.a $(INTERNAL_LIBS)
 	@echo "    [CC] $@"
 	$(Q) $(CC) $(CFLAGS) $(LDFLAGS) -static -L$(interfacebuilddir) -o $@ \
-	      $(OBJECTS) lvm-static.o $(LVMLIBS) $(STATIC_LIBS) $(LIBS)
+	      $(OBJECTS) lvm-static.o $(LVMLIBS) $(STATIC_LIBS) $(LIBS) $(top_builddir)/lib/liblvm-internal.a -ldevmapper -luuid -pthread -L../base -lbase -lm $(INTERNAL_LIBS)
 
 liblvm2cmd.a: $(top_builddir)/lib/liblvm-internal.a $(OBJECTS) lvmcmdlib.o lvm2cmd.o
 	@echo "    [AR] $@"
diff -up LVM2.2.03.01/make.tmpl.in.omv~ LVM2.2.03.01/make.tmpl.in
--- LVM2.2.03.01/make.tmpl.in.omv~	2018-11-07 20:23:05.013284321 +0100
+++ LVM2.2.03.01/make.tmpl.in	2018-11-07 20:23:15.033334557 +0100
@@ -122,6 +122,8 @@ systemd_dir = $(DESTDIR)@systemdutildir@
 tmpfiles_dir = $(DESTDIR)@tmpfilesdir@
 ocf_scriptdir = $(DESTDIR)@OCFDIR@
 pythonprefix = $(DESTDIR)$(prefix)
+interface = @interface@
+interfacebuilddir = $(top_builddir)/libdm/$(interface)
 
 # N.B. No $(DESTDIR) prefix here.
 python2dir = @PYTHON2DIR@
