diff --git a/libdm/dm-tools/Makefile.in b/libdm/dm-tools/Makefile.in
index 44f19e7..f913fec 100644
--- a/libdm/dm-tools/Makefile.in
+++ b/libdm/dm-tools/Makefile.in
@@ -48,7 +48,7 @@ include $(top_builddir)/libdm/make.tmpl
 CFLAGS_dmsetup.o += $(UDEV_CFLAGS) $(EXTRA_EXEC_CFLAGS)
 CFLAGS_dmfilemapd.o += $(EXTRA_EXEC_CFLAGS)
 LDFLAGS += -L$(interfacebuilddir)
-DMLIBS = -ldevmapper
+DMLIBS = -ldevmapper -lm
 
 dmsetup: dmsetup.o
 	@echo "    [CC] $@"
