diff -Naur LVM2.2.03.03/conf/example.conf.in LVM2.2.03.03.tpg/conf/example.conf.in
--- LVM2.2.03.03/conf/example.conf.in	2019-06-07 15:26:46.000000000 +0000
+++ LVM2.2.03.03.tpg/conf/example.conf.in	2019-06-10 15:51:28.993042574 +0000
@@ -106,7 +106,7 @@
 	# Example
 	# preferred_names = [ "^/dev/mpath/", "^/dev/mapper/mpath", "^/dev/[hs]d" ]
 	# 
-	# This configuration option does not have a default value defined.
+	preferred_names = [ "^/dev/mpath/", "^/dev/mapper/mpath", "^/dev/[hs]d" ]
 
 	# Configuration option devices/filter.
 	# Limit the block devices that are used by LVM commands.
diff -Naur LVM2.2.03.03/lib/config/config_settings.h LVM2.2.03.03.tpg/lib/config/config_settings.h
--- LVM2.2.03.03/lib/config/config_settings.h	2019-06-07 15:26:46.000000000 +0000
+++ LVM2.2.03.03.tpg/lib/config/config_settings.h	2019-06-10 15:54:06.938055370 +0000
@@ -269,7 +269,7 @@
 	"    Use no hints.\n"
 	"#\n")
 
-cfg_array(devices_preferred_names_CFG, "preferred_names", devices_CFG_SECTION, CFG_ALLOW_EMPTY | CFG_DEFAULT_UNDEFINED , CFG_TYPE_STRING, NULL, vsn(1, 2, 19), NULL, 0, NULL,
+cfg_array(devices_preferred_names_CFG, "preferred_names", devices_CFG_SECTION, CFG_ALLOW_EMPTY, CFG_TYPE_STRING, "#S^/dev/mpath/#S^/dev/mapper/mpath#S^/dev/[hs]d", vsn(1, 2, 19), NULL, 0, NULL,
 	"Select which path name to display for a block device.\n"
 	"If multiple path names exist for a block device, and LVM needs to\n"
 	"display a name for the device, the path names are matched against\n"
