diff -up LVM2.2.03.01/make.tmpl.in.omv~ LVM2.2.03.01/make.tmpl.in
--- LVM2.2.03.01/make.tmpl.in.omv~	2018-11-07 19:15:32.368410312 +0100
+++ LVM2.2.03.01/make.tmpl.in	2018-11-07 19:15:37.531435867 +0100
@@ -64,7 +64,7 @@ PYCOMPILE = $(top_srcdir)/autoconf/py-co
 LIBS = @LIBS@
 LIBS += $(SELINUX_LIBS) $(UDEV_LIBS) $(BLKID_LIBS) $(RT_LIBS) -lm
 # Extra libraries always linked with static binaries
-STATIC_LIBS = $(SELINUX_LIBS) $(UDEV_LIBS) $(BLKID_LIBS)
+STATIC_LIBS = $(SELINUX_LIBS) $(UDEV_LIBS) $(BLKID_LIBS) -pthread -luuid
 DEFS += @DEFS@
 # FIXME set this only where it's needed, not globally?
 CFLAGS ?= @COPTIMISE_FLAG@ @CFLAGS@
