diff -p -up ./man/vgmknodes.8.in.vgmknodes-man ./man/vgmknodes.8.in
--- ./man/vgmknodes.8.in.vgmknodes-man	2012-08-07 21:05:15.000000000 +0000
+++ ./man/vgmknodes.8.in	2012-09-11 09:43:01.309335983 +0000
@@ -5,6 +5,7 @@ vgmknodes \- recreate volume group direc
 .B vgmknodes
 .RB [ \-d | \-\-debug ]
 .RB [ \-h | \-\-help ]
+.RB [ \-\-ignorelockingfailure]
 .RB [ \-\-refresh ]
 .RB [ \-v | \-\-verbose ]
 .RI [[ VolumeGroupName | LogicalVolumePath ]...]
