--- LVM2.2.02.58/conf/example.conf.in	2010-01-07 19:54:21.000000000 +0000
+++ LVM2.2.02.58-new/conf/example.conf.in	2010-01-20 18:30:32.000000000 +0000
@@ -58,7 +58,7 @@ devices {
     # preferred_names = [ ]
 
     # Try to avoid using undescriptive /dev/dm-N names, if present.
-    # preferred_names = [ "^/dev/mpath/", "^/dev/mapper/mpath", "^/dev/[hs]d" ]
+    preferred_names = [ "^/dev/mpath/", "^/dev/mapper/mpath", "^/dev/[hs]d" ]
 
     # In case no prefererred name matches or if preferred_names are not
     # defined at all, builtin rules are used to determine the preference.
