diff -Naur LVM2.2.03.02/libdm/Makefile.in LVM2.2.03.02.tpg/libdm/Makefile.in
--- LVM2.2.03.02/libdm/Makefile.in	2018-12-18 14:22:34.000000000 +0000
+++ LVM2.2.03.02.tpg/libdm/Makefile.in	2019-04-15 11:12:32.326000000 +0000
@@ -42,11 +42,12 @@
 
 ifeq ("@STATIC_LINK@", "yes")
 LIB_STATIC = $(interface)/libdevmapper.a
-endif
-
+TARGETS = libdevmapper.a
+else
 LIB_SHARED = $(interface)/libdevmapper.$(LIB_SUFFIX)
 LIB_VERSION = $(LIB_VERSION_DM)
 TARGETS = libdevmapper.$(LIB_SUFFIX) libdevmapper.$(LIB_SUFFIX).$(LIB_VERSION)
+endif
 
 CFLOW_LIST = $(SOURCES)
 CFLOW_LIST_TARGET = libdevmapper.cflow
@@ -68,6 +69,9 @@
 	@echo "    [LN] $<"
 	$(Q) $(LN_S) -f $< $@
 
+libdevmapper.a: $(LIB_STATIC)
+	$(LN_S) -f $< $@
+
 .PHONY: install_dynamic install_static install_include \
 	install_ioctl install_ioctl_static \
 	install_pkgconfig
diff -Naur LVM2.2.03.02/libdm/dm-tools/Makefile.in LVM2.2.03.02.tpg/libdm/dm-tools/Makefile.in
--- LVM2.2.03.02/libdm/dm-tools/Makefile.in	2018-12-18 14:22:34.000000000 +0000
+++ LVM2.2.03.02.tpg/libdm/dm-tools/Makefile.in	2019-04-15 11:10:59.443000000 +0000
@@ -58,7 +58,7 @@
 dmsetup.static: dmsetup.o
 	@echo "    [CC] $@"
 	$(Q) $(CC) $(CFLAGS) $(LDFLAGS) -static \
-	      -o $@ $+ $(DMLIBS) $(STATIC_LIBS)
+	      -o $@ $+ $(DMLIBS) $(STATIC_LIBS) -L$(top_builddir)/libdm -ldevmapper -lm -pthread
 
 install_dmsetup_dynamic: dmsetup
 	@echo "    [INSTALL] $<"
diff -Naur LVM2.2.03.02/make.tmpl.in LVM2.2.03.02.tpg/make.tmpl.in
--- LVM2.2.03.02/make.tmpl.in	2018-12-18 14:22:34.000000000 +0000
+++ LVM2.2.03.02.tpg/make.tmpl.in	2019-04-15 11:13:38.510000000 +0000
@@ -118,6 +118,8 @@
 tmpfiles_dir = $(DESTDIR)@tmpfilesdir@
 ocf_scriptdir = $(DESTDIR)@OCFDIR@
 pythonprefix = $(DESTDIR)$(prefix)
+interface = @interface@
+interfacebuilddir = $(top_builddir)/libdm/$(interface)
 
 # N.B. No $(DESTDIR) prefix here.
 python2dir = @PYTHON2DIR@
diff -Naur LVM2.2.03.02/tools/Makefile.in LVM2.2.03.02.tpg/tools/Makefile.in
--- LVM2.2.03.02/tools/Makefile.in	2018-12-18 14:22:34.000000000 +0000
+++ LVM2.2.03.02.tpg/tools/Makefile.in	2019-04-15 11:13:15.306000000 +0000
@@ -137,7 +137,7 @@
 lvm.static: $(OBJECTS) lvm-static.o $(LVMINTERNAL_LIBS)
 	@echo "    [CC] $@"
 	$(Q) $(CC) $(CFLAGS) $(LDFLAGS) -static -L$(interfacebuilddir) -o $@ $+ \
-	      $(DMEVENT_LIBS) $(STATIC_LIBS) $(LVMLIBS)
+	      $(DMEVENT_LIBS) $(STATIC_LIBS) $(LVMLIBS) $(top_builddir)/lib/liblvm-internal.a -ldevmapper -luuid -pthread -L../base -lbase -lm $(INTERNAL_LIBS)
 
 liblvm2cmd.a: $(top_builddir)/lib/liblvm-internal.a $(OBJECTS) lvmcmdlib.o lvm2cmd.o
 	@echo "    [AR] $@"
