diff --git a/conf/example.conf.in b/conf/example.conf.in
index a26066a..ef9bb5c 100644
--- a/conf/example.conf.in
+++ b/conf/example.conf.in
@@ -101,7 +101,7 @@ devices {
 	# Rule 3 prefers a symlink.
 	# Rule 4 prefers the path with least value in lexicographical order.
 	# Example:
-	# preferred_names = [ "^/dev/mpath/", "^/dev/mapper/mpath", "^/dev/[hs]d" ]
+	preferred_names = [ "^/dev/mpath/", "^/dev/mapper/mpath", "^/dev/[hs]d" ]
 	# This configuration option does not have a default value defined.
 
 	# Configuration option devices/filter.
