diff -up LVM2.2.03.43/man/lvm.8_main.orig LVM2.2.03.43/man/lvm.8_main
--- LVM2.2.03.43/man/lvm.8_main.orig
+++ LVM2.2.03.43/man/lvm.8_main
@@ -37,8 +37,8 @@ that command.
 On invocation, \fBlvm\fP requires that only the standard file descriptors
 stdin, stdout and stderr are available.  If others are found, they are
 closed and a warning about the leak is issued.
-This warning can be suppressed by setting the environment variable
-.BR LVM_SUPPRESS_FD_WARNINGS .
+By default warnings are suppressed and can be enabled by setting the environment variable
+.BR LVM_SHOW_FD_WARNINGS .
 .P
 Where commands take VG or LV names as arguments, the full path name is
 optional.  An LV called "lvol0" in a VG called "vg0" can be specified
diff -up LVM2.2.03.43/tools/lvmcmdline.c.orig LVM2.2.03.43/tools/lvmcmdline.c
--- LVM2.2.03.43/tools/lvmcmdline.c.orig
+++ LVM2.2.03.43/tools/lvmcmdline.c
@@ -3592,7 +3592,7 @@ int lvm2_main(int argc, char **argv)
 	if (!_get_custom_fds(&custom_fds))
 		return EINIT_FAILED;
 
-	if (!daemon_close_stray_fds(base, getenv("LVM_SUPPRESS_FD_WARNINGS") ? 1 : 0,
+	if (!daemon_close_stray_fds(base, getenv("LVM_SHOW_FD_WARNINGS") ? 0 : 1,
 				    STDERR_FILENO, &custom_fds))
 		return EINIT_FAILED;
 
