diff -Naur LVM2.2.03.13/man/lvm.8_main LVM2.2.03.13.tpg/man/lvm.8_main
--- LVM2.2.03.13/man/lvm.8_main	2021-08-11 15:37:43.000000000 +0000
+++ LVM2.2.03.13.tpg/man/lvm.8_main	2021-10-06 21:21:33.557977644 +0000
@@ -37,8 +37,8 @@
 On invocation, \fBlvm\fP requires that only the standard file descriptors
 stdin, stdout and stderr are available.  If others are found, they
 get closed and messages are issued warning about the leak.
-This warning can be suppressed by setting the environment variable
-.BR LVM_SUPPRESS_FD_WARNINGS .
+By default warnings are suppressed and can be enabled by setting the environment variable
+.BR LVM_SHOW_FD_WARNINGS .
 .P
 Where commands take VG or LV names as arguments, the full path name is
 optional.  An LV called "lvol0" in a VG called "vg0" can be specified
diff -Naur LVM2.2.03.13/tools/lvmcmdline.c LVM2.2.03.13.tpg/tools/lvmcmdline.c
--- LVM2.2.03.13/tools/lvmcmdline.c	2021-08-11 15:37:43.000000000 +0000
+++ LVM2.2.03.13.tpg/tools/lvmcmdline.c	2021-10-06 21:19:24.398825553 +0000
@@ -3498,7 +3498,9 @@
 	}
 #endif
 
-	if (getenv("LVM_SUPPRESS_FD_WARNINGS"))
+	if (getenv("LVM_SHOW_FD_WARNINGS"))
+		suppress_warnings = 0;
+	else
 		suppress_warnings = 1;
 
 	if (!(d = opendir(_fd_dir))) {
