diff -Naur LVM2.2.02.182/libdaemon/server/daemon-server.c LVM2.2.02.182.tpg/libdaemon/server/daemon-server.c
--- LVM2.2.02.182/libdaemon/server/daemon-server.c	2018-10-30 09:13:42.000000000 +0000
+++ LVM2.2.02.182.tpg/libdaemon/server/daemon-server.c	2018-11-07 20:32:04.394983988 +0000
@@ -30,6 +30,7 @@
 #include <sys/un.h>
 #include <unistd.h>
 #include <signal.h>
+#include <malloc.h>
 
 #include <syslog.h> /* FIXME. For the global closelog(). */
 
@@ -645,6 +646,7 @@
 
 	while (!failed) {
 		_reset_timeout(s);
+		malloc_trim(0);
 		FD_ZERO(&in);
 		FD_SET(s.socket_fd, &in);
 		if (select(FD_SETSIZE, &in, NULL, NULL, _get_timeout(s)) < 0 && errno != EINTR)
