diff -up LVM2.2.03.01/libdaemon/server/daemon-server.c.4~ LVM2.2.03.01/libdaemon/server/daemon-server.c
--- LVM2.2.03.01/libdaemon/server/daemon-server.c.4~	2018-10-31 15:54:24.000000000 +0100
+++ LVM2.2.03.01/libdaemon/server/daemon-server.c	2018-11-07 19:18:38.525506313 +0100
@@ -30,6 +30,7 @@
 #include <sys/un.h>
 #include <unistd.h>
 #include <signal.h>
+#include <malloc.h>
 
 #include <syslog.h> /* FIXME. For the global closelog(). */
 
@@ -645,6 +646,7 @@ void daemon_start(daemon_state s)
 
 	while (!failed) {
 		_reset_timeout(s);
+		malloc_trim(0);
 		FD_ZERO(&in);
 		FD_SET(s.socket_fd, &in);
 		if (select(FD_SETSIZE, &in, NULL, NULL, _get_timeout(s)) < 0 && errno != EINTR)
