diff -Naur LVM2.2.03.13/libdaemon/server/daemon-server.c LVM2.2.03.13.tpg/libdaemon/server/daemon-server.c
--- LVM2.2.03.13/libdaemon/server/daemon-server.c	2021-08-11 15:37:43.000000000 +0000
+++ LVM2.2.03.13.tpg/libdaemon/server/daemon-server.c	2021-10-06 21:13:07.050513376 +0000
@@ -28,6 +28,7 @@
 #include <sys/un.h>
 #include <unistd.h>
 #include <signal.h>
+#include <malloc.h>
 
 #include <syslog.h> /* FIXME. For the global closelog(). */
 
@@ -666,6 +667,7 @@
 
 	while (!failed && !_shutdown_requested) {
 		_reset_timeout(s);
+		malloc_trim(0);
 		FD_ZERO(&in);
 		FD_SET(s.socket_fd, &in);
 
